pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiStageEngineIntegrationTest.java

```diff
@@ -327,6 +327,24 @@ public void testMultiValueColumnAggregationQuery(boolean useMultiStageQueryEngin
     Assert.assertTrue(jsonNode.get("resultTable").get("rows").get(0).get(0).asDouble() < 17000);
   }
 
+  @Test(dataProvider = "useBothQueryEngines")
```

Comment:
Why is this testing both query engines if it is in the MSE's integration test class?